src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/.template.config/template.json

```diff
@@ -106,11 +103,19 @@
         }
       ]
     },
-    "IsAzureOpenAi": {
+    "UseManagedIdentity": {
+      "type": "parameter",
+      "displayName": "Use keyless authentication for Azure services",
+      "datatype": "bool",
+      "defaultValue": "true",
+      "isEnabled": "(AiServiceProvider == \"azureopenai\" || AiServiceProvider == \"azureaifoundry\" || VectorStore == \"azureaisearch\")",
```

Comment:
This expression seems to duplicate the `UseAzure` symbol below. Could these be merged somehow?